scripts/run_examples.py

```diff
@@ -182,7 +189,12 @@ def run_examples(keep_artifacts=None):
                         # the tempdir cleanup later
                         f"/S _?={env_dir}"
```

Comment:
Idea: maybe this suffers from the same problem as `/D` during the install (line 141 or so?).

Let's try with:

```suggestion
                        "/S", *f"_?={env_dir}".split()
```